scipy/optimize/_minimize.py

```diff
@@ -205,9 +208,17 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
     and either the Hessian or a function that computes the product of
     the Hessian with a given vector. Suitable for large-scale problems.
 
+    Method :ref:`trust-krylov <optimize.minimize-trustkrylov>` uses
+    the Newton GLTR trust-region algorithm [13]_ for unconstrained
+    minimization. This algorithm requires the gradient
+    and either the Hessian or a function that computes the product of
+    the Hessian with a given vector. Suitable for large-scale problems.
+    On indefinite problems it requires usually less iterations than the
+    `trust-ncg` method and is recommended for medium and large-scale problems.
+
     Method :ref:`trust-region-exact <optimize.minimize-trustexact>`
     is a trust-region method for unconstrained minimization in which
-    quadratic subproblems are solved almost exactly [13]_. This
+    quadratic subproblems are solved almost exactly [14]_, [15]_. This
```

Comment:
@antonior92 Thanks for spotting this, I have corrected this. Sorry for the merge from `master` again, I just realized that this is due to the github resolve confllicts to base branch GUI...